### PR TITLE
Update kubed sync example and use secret template instead of pre-annotated secret

### PR DESCRIPTION
**Pull request by AljoschaP**
Because the use of pre-annotated secrets for syncing certificates across namespaces is error-prone and `secretTemplate` is available since [1.5.0](https://github.com/jetstack/cert-manager/pull/3828), this PR updates the kubed example for syncing secrets to use `secretTemplate` instead of pre-created secrets.

### CODE REVIEW

File: content/en/docs/faq/kubed.md [hunk 1]
```diff
@@ -73,3 +73,5 @@ spec:
     kind: Issuer
     group: cert-manager.io
 ```
+
+[CertificateSecretTemplate]: ../../reference/api-docs/#cert-manager.io/v1alpha3.CertificateSecretTemplate
```
Comment (irbekrm) [inline content/en/docs/faq/kubed.md:65]: We should link to the `v1` API version as the others are deprecated.

**Review by irbekrm — CHANGES_REQUESTED** [review]
Hi @AljoschaP,

Thank you very much for the contribution!

I've added a minor nit about linking to v1 API version, apart from that this is a very useful change 👍🏼 

/ok-to-test

File: content/en/docs/faq/kubed.md (hunk 1 above)
Comment (AljoschaP) [reply content/en/docs/faq/kubed.md]: Thanks for the review. I've updated the link.

**Comment by irbekrm** [comment]
Thanks for making the changes! This now looks good to me 👍🏼 

Just needs the last three commits to be signed to get passed the CI tests 😅

**Comment by AljoschaP** [comment]
> Thanks for making the changes! This now looks good to me 👍🏼
> 
> Just needs the last three commits to be signed to get passed the CI tests sweat_smile



> Thanks for making the changes! This now looks good to me 👍🏼
> 
> Just needs the last three commits to be signed to get passed the CI tests sweat_smile

Should be okay now :smiley: Thank you :+1:

**Comment by irbekrm** [comment]
Thanks for working on this!

/lgtm
/approve

**Pull request merged**